src/bokeh/io/webdriver.py

```diff
@@ -62,11 +63,17 @@ def create_firefox_webdriver(scale_factor: float = 1) -> WebDriver:
     if geckodriver is None:
         raise RuntimeError("geckodriver is not installed or not present on PATH")
 
+    import selenium
     from selenium.webdriver.firefox.options import Options
     from selenium.webdriver.firefox.service import Service
     from selenium.webdriver.firefox.webdriver import WebDriver as Firefox
 
-    service = Service(log_path=devnull)
+    if Version(selenium.__version__) >= Version("4.11"):  # type: ignore[attr-defined]
```

Comment:
Instead of ignoring type errors, you could define `__version__: str` in `bokeh/src/typings/selenium/__init__.pyi`.